client/jetpack-cloud/sections/agency-dashboard/sites-overview/sites-dashboard-v2.tsx

```diff
@@ -129,6 +129,19 @@ export default function SitesDashboardV2() {
 		updateDashboardURLQueryArgs( { search: sitesViewState.search } );
 	}, [ sitesViewState.search ] );
 
+	// Set or clear filter depending on sites submenu path selected
+	useEffect( () => {
+		if ( path === '/sites' || path === '/sites/favorites' ) {
+			setSitesViewState( { ...sitesViewState, filters: [] } );
+		}
+		if ( path === '/sites?issue_types=all_issues' ) {
+			setSitesViewState( {
+				...sitesViewState,
+				filters: [ { field: 'status', operator: 'in', value: 1 } ],
```

Comment:
Maybe:

```suggestion
				filters: [ { field: 'status', operator: 'in', value: 1 } ],
				search: '',
```

Reply:
❤️

Reply:
Note that the value of item 1 might change, so this is a bit of a magic number... 😅

Also, if "Needs attention" had its own route, there'd be no need for a special condition here.

Reply:
Yep, that is inherited from DataViews; they use magic numbers for referencing to filters.